docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Create a multi-zone universe using Yugabyte Platform and Kubernetes
+headerTitle: Create a multi-zone universe with Kubernetes
+linkTitle: Multi-zone universe
+description: Use Yugabyte Platform and Kubernetes to create a YugabyteDB universe that spans multiple availability zones.
+menu:
+  latest:
+    identifier: create-multi-zone-universe-kubernetes
+    parent: create-deployments
+    weight: 20
+isTocNested: true
+showAsideToc: true
+---
+
+  <ul class="nav nav-tabs-alt nav-tabs-yb">
+
+<li>
+    <a href="/latest/yugabyte-platform/create-deployments/create-universe-multi-zone" class="nav-link">
+      <i class="fas fa-building" aria-hidden="true"></i>
+Generic</a>
+  </li>
+
+  <li>
+    <a href="/latest/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes" class="nav-link active">
+      <i class="fas fa-cubes" aria-hidden="true"></i>
+      Kubernetes
+    </a>
+  </li>
+
+</ul>
+
+This section describes how to create a YugabyteDB universe in one geographic region across multiple availability zones using Kubernetes as a cloud provider. 
+
+## Prerequisites
+
+Before you start creating a universe, ensure that you performed steps described in [Configure the Kubernetes cloud provider](https://docs.yugabyte.com/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes/). The following illustration shows the **Managed Kubernetes Service configs** page that you should be able to see if you use Yugabyte Platform to navigate to **Configs > Cloud Provider Configuration > Infrastructure > Managed Kubernetes Service**:
+
+![img](/images/yb-platform/kubernetes-config1.png)
+
+Note the following: 
+
+- Yugabyte Platform version 2.7.1.1 is used in examples throughout this document.
```

Comment:
We no longer need to mention the version after updating all the images in review comments.

Reply:
Deleted the sentence with the version.